src/ServiceControl/Operations/CheckRavenDBIndexLag.cs

```diff
@@ -0,0 +1,90 @@
+﻿namespace ServiceControl
+{
+    using System;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using NServiceBus.CustomChecks;
+    using NServiceBus.Logging;
+    using Raven.Client;
+    using Raven.Abstractions.Data;
+    using ServiceBus.Management.Infrastructure.Settings;
+
+    class CheckRavenDBIndexLag : CustomCheck
```

Comment:
I think all of the same comments applied to the other custom check likely apply to this one. :)